site/server/conf/evolutions/default/1.sql

```diff
@@ -1,15 +1,15 @@
 # --- !Ups
 
-CREATE TABLE User (
-    id bigint primary key auto_increment,
-    login varchar(255) unique not null,
-    name varchar(255) not null,
-    githubId varchar(255) not null,
-    pictureUrl varchar(255) not null,
-    githubUrl varchar(255) not null,
-    email varchar(255) not null
+CREATE TABLE users (
+    id bigserial primary key,
+    login text UNIQUE NOT NULL,
```

Comment:
it doesn't. It only complains when weren't defined as 'NOT NULL'. Tests are passing successfully.